### DISCUSSION TITLE
Default Arsenal Sorting "by Amount"

### DISCUSSION THREAD

**Discussion by Holgerhollebolle** [acemod/ACE3#9518] [Ideas]
95% of the Times i open the Arsenal and click on "sort by Amount". 5% of the Times I might use "by Mod". 
It is very annoying that I have to do this every time I open the Arsenal. I can imagine the most of the People do it that way as I see not a reason why you would want to sort Alphabetically (by Amount also sorts Alphabetically) or by Weight...these are really edgecases.

Please do sort by Amount by default. This would make the life of many people easier.

**Comment by Holgerhollebolle** [comment]
btw...I am only talking about the Right side of the Arsenal.

**Comment by Andx667** [comment]
I only ever sort by Mod. Don't take your use case as the default for everyone.

Of course the best solution would be a setting so everyone can set the default to whatever they prefer.

**Reply by Holgerhollebolle** [reply]
I only said I can imagine most people want to sort by amount...but an option would be of course the best solution.

**Comment by Drofseh** [comment]
#3594
